src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/runWithEnvVariables.groovy

```diff
@@ -25,6 +26,7 @@ podTemplate(
         echo OUTSIDE_CONTAINER_ENV_VAR_FROM_SECRET = $CONTAINER_ENV_VAR_FROM_SECRET or `echo $CONTAINER_ENV_VAR_FROM_SECRET | tr [a-z] [A-Z]`
         echo OUTSIDE_POD_ENV_VAR = $POD_ENV_VAR
         echo OUTSIDE_POD_ENV_VAR_FROM_SECRET = $POD_ENV_VAR_FROM_SECRET or `echo $POD_ENV_VAR_FROM_SECRET | tr [a-z] [A-Z]`
+        echo OUTSIDE_EMPTY_POD_ENV_VAR_FROM_SECRET = '$EMPTY_POD_ENV_VAR_FROM_SECRET '
```

Comment:
I think you meant

```suggestion
        echo "OUTSIDE_EMPTY_POD_ENV_VAR_FROM_SECRET = '$EMPTY_POD_ENV_VAR_FROM_SECRET'"
```

or

```suggestion
        echo OUTSIDE_EMPTY_POD_ENV_VAR_FROM_SECRET = "'$EMPTY_POD_ENV_VAR_FROM_SECRET'"
```

as what you have will print

```
OUTSIDE_EMPTY_POD_ENV_VAR_FROM_SECRET = '$EMPTY_POD_ENV_VAR_FROM_SECRET '
```

with no interpolation.